AmericanMedicalAssociation/ama-style-guide-2
# EWL-8007: Redesign topic terms

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Jira Ticket**
- [EWL-8007: Article Page Redesign - Topic Terms](https://issues.ama-assn.org/browse/EWL-8007)

## Description
* Changing tags structure per redesign
* Changing Tags Tags markup structure
* Changing styles per redesign and adding responsive styles.


## To Test
- [ ] Generate the styleguide `gulp serve`
- Go to  ` Molecules >> Tags` and ensure is displaying like in the Zepling designs attached to the ticker
- Go to Pages and ensure Tags are displaying properly for the following pages:
  * News Article Page Template
  * Event Detail Template
  * People Bio Template
  * People Listing Template
  * Press Release Template
  * Resource Page template## Visual Regressions


## Relevant Screenshots/GIFs
<img width="649" alt="Screen Shot 2020-06-12 at 4 07 15 PM" src="https://user-images.githubusercontent.com/5325044/84546600-4e478200-acc7-11ea-983c-a9bbec1901e3.png">
![Screen Shot 2020-06-12 at 4 10 39 PM](https://user-images.githubusercontent.com/5325044/84546604-4f78af00-acc7-11ea-859e-526220bfa608.png)


## Remaining Tasks
Things relevant to here may be updating [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode) with the latest naming changes or review of the relative [D8](https://github.com/AmericanMedicalAssociation/ama-d8) work.
![Screen Shot 2020-06-12 at 4 10 49 PM](https://user-images.githubusercontent.com/5325044/84546607-50a9dc00-acc7-11ea-9e77-be1dc7bb537b.png)



## Additional Notes
The integration in the CMS depends of this PR https://github.com/AmericanMedicalAssociation/ama-d8/pull/2045

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
